apps/ejabberd/src/mongoose_commands.erl

```diff
@@ -230,17 +230,17 @@ list(U) ->
     list(U, any, any, any).
 
 %% @doc List commands, available for this user, filtered by category.
--spec list(caller(), atom()) -> [t()].
+-spec list(caller(), binary() | any) -> [t()].
```

Comment:
@bartekgorny, `any` as atom? Do we expect atom `any` as value for the `C` parameter?